src/components/tag.scss

```diff
@@ -0,0 +1,87 @@
+%tag-global {
+  // Public API (allowed to be overridden)
+  --_op-tag-background-color: var(--op-color-primary-base);
+  --_op-tag-color: var(--op-color-primary-on-base);
+  --_op-tag-background-color-hover: var(--op-color-primary-minus-one);
+  --_op-tag-color-hover: var(--op-color-primary-on-minus-one);
+  --_op-tag-icon-background-color-hover: var(--op-color-primary-minus-three);
+  --_op-tag-icon-color-hover: var(--op-color-primary-on-minus-three);
+
+  display: inline-flex;
+  align-items: center;
+  padding: var(--op-space-2x-small) var(--op-space-x-small);
+  border-radius: var(--op-radius-pill);
+  background-color: var(--_op-tag-background-color);
+  color: var(--_op-tag-color);
+  font-size: var(--op-font-x-small);
+  font-weight: var(--op-font-weight-bold);
+  gap: var(--op-space-2x-small);
+
+  &:hover {
+    background-color: var(--_op-tag-background-color-hover);
+  }
+
+  .material-symbols-outlined {
+    border-radius: var(--op-radius-pill);
+
+    &:hover {
+      background-color: var(--_op-tag-icon-background-color-hover);
+      color: var(--_op-tag-icon-color-hover);
+      cursor: pointer;
+    }
+  }
+}
+
+.tag--default {
+  @extend %tag-global;
+}
+
+.tag--readonly {
```

Comment:
Might be helpful to have a write up of the distinction between Variants (btn, btn-primary, btn-delete, etc) vs Modifiers (btn--pill, btn--icon, etc). Variants are top level components that are distinct. Modifiers are stylistic tweaks that apply to all variants.

Reply:
Could be good to capture any other missing documentation or concept clarification you have come across.